design/enable-selinux-mount.md

```diff
@@ -0,0 +1,84 @@
+---
+title: SELinux Mount Support for LocalPV-ZFS
+authors:
+  - "@cjorge-graphops"
+creation-date: 2024-11-01
+last-updated: 2024-11-01
+---
```

Comment:
@tiagolobocastro Should this be here?

Reply:
You mean vs the root openebs? Seems we already have pv-migration here in this repo as well.
So basically, going forward do we want to keep designs per repo or on umbrella repo? CC @avishnu

Reply:
Let's just keep it here for now.
Let's discuss on the next maintainers call on how to organize these better.